src/handleUrl.js

```diff
@@ -0,0 +1,35 @@
+import globby from 'globby';
+
+export default async function(assets) {
+  const globbyAssets = [];
+  const normalAssets = [];
+  try {
```

Comment:
If  filepath is null or undefined. HasMagic will throw an error. Maybe I should do something juadgement~